packages/core-mobile/e2e/tests/updateAppVersion/loginWithOlderVersion.sh

```diff
@@ -0,0 +1,18 @@
+set -o pipefail
+if [ "$IS_REGRESSION_RUN" = true ]; then
+
+  npm rebuild detox
+  adb uninstall com.avaxwallet
+  npx ts-node ./e2e/tests/updateAppVersion/getOldAndroidVersion.js
+
+  adb install -r "./e2e/tests/updateAppVersion/oldVersionApk/app-external-e2e-bitrise-signed.apk"
```

Comment:
Looks like  `_install-old-android-version`, `_install-latest-android-version` is added to pixel6  and pixel6 is running both internal and external build testing. the internal smoke testing has still access to external build?